docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -375,6 +375,20 @@ Return value:
 
 ### eBPF Maps
 
+#### `newMap(name string, typ uint32, keySize uint32, valueSize uint32, maxEntries uint32) uint32`
+
+Create a new eBPF map.
```

Comment:
What is the main use case for this? Will you need to add types (BTF) to the maps, or will we only need untyped maps?